src/models/preHooks/validate.js

```diff
@@ -14,6 +15,23 @@ module.exports = {
     if (!hasCompany && !isPopulate && !isVendorUser && !requestingOwnInfos && !allCompanies) next(Boom.badRequest());
     next();
   },
+  formatQuery(next) {
+    const query = this.getQuery();
+    const flattenQuery = {};
+    for (const entry of Object.entries(query)) {
+      if (entry[1] && typeof entry[1] === 'object' && !Array.isArray(entry[1]) && Object.keys(entry[1])[0] &&
```

Comment:
Je ne comprends pas bien cette partie du code mais il me semble que peux utiliser directement `const [key, value] of Object.entries(query)` pour être plus compréhensible

Reply:
bien vu !